.github/workflows/continuous-integration.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [18.x, 20.x]
```

Comment:
18.x?

Reply:
I added support for 20. That includes support for 18. So why not 🤷?